### ISSUE TITLE
Is this a bug? Bouncing/dancing nameplates

### ISSUE THREAD

**Issue by KGan** [Tercioo/Plater-Nameplates#157]
**Game and Plater version**
Plater-v552-Retail - DF v415 - 10.0.5

**Describe the bug**
- Steps to Reproduce (what did you do to make the bug happen):
  stand anywhere with nameplates nearing any boundary of the screen (typically upper)

- Result (what happens when you follow the steps above):
-  https://media.giphy.com/media/olBf5BCoqnB3CV9eiI/giphy.gif
- plates bounce to completely random order constantly

- Expected (what you think the expected behavior would be when following the steps):
 There should be a stable or deterministic equilibrium for where the plates are at least if 1. the camera is not moving and 2. none of the enemies are moving

- When the bug started (a date, wow path release, details update, etc)?
 since I started playing the game recently, whatever the patch before 10.0.5 was but didn't bother to report until I started doing content where this affected my ability to target things

**Lua Error**
none

**Comment by cont1nuity** [comment]
All nameplate movement is controlled by blizzard. Players (and addons) only have a very limited set of options to influence this. Plater offers these options on the advanced tab through the overlap, movement speed and selection space options.

TL;DR: blizzard. Can't do much about it.
